clickhouse/base/string_utils.h

```diff
@@ -23,4 +26,37 @@ inline T FromString(const StringView& s) {
    return result;
 }
 
+inline const char* UUIDFormat() {
```

Comment:
Also, since `UUIDFormat()`, `ToString(UUID)`, `FromString(..., UUID)` are used only in unit-tests, it makes sense to put those into the files that are used exclusively by test binary, not to the library headers.

Reply:
I think, this functions are pretty useful for library users.  

If someone needs to pass UUID as query params, the function `ToString(UUID) ` will be extremely useful.

Reply:
There are approximately **0** settings that are UUID. Users are most likely to don't use UUIDs string representation or have their own string serialization/deserialzation for UUIDs.
So usability of those functions is questionable at best.

Please move those functions to `ut`.